src/css/base.scss

```diff
@@ -86,47 +95,38 @@ nav {
   background-image: linear-gradient(white, #e2e2e2);
   color: #2b8b9e;
   font-size: $bodySizeSmall;
-  height: 70px;
+  height: 4em;
   justify-content: center;
   width: 100%;
     h1 {
-      font-size: 3.3em;
+      font-size: 4em;
+      font-family: $feature-font;
+      margin-top: 1em;
     }
-    h6 {
-      font-size: $bodySizeMedium;
+    h2 {
+      font-size: $bodySizeLarge;
     }
 }
 
-nav div {
-  @extend %flex_center-shared;
-  justify-content: space-between;
-  width: 1100px;
-}
-
 #welcome-name {
-  color: #f0a83d;
+  color: $accent_light;
+  font-family: $feature-font;
 }
 
 header {
-  @extend %flex_center-shared;
+  display: flex;
   justify-content: space-between;
-  height: max-content;
   padding: 0 50px;
-  width: 1200px;
-    h2 {
-      color: $accent_color;
-      font-size: 2.5em;
-      margin: 3px 0 0 0;
-    }
+  width: 100%;
```

Comment:
Good use of percentage for width and all the other refactoring you have done! This will make the page much more responsive.